manifests/f/FilesCommunity/Files/3.3.0.0/FilesCommunity.Files.installer.yaml

```diff
@@ -0,0 +1,23 @@
+# Created with komac v2.2.1
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.6.0.schema.json
+
+PackageIdentifier: FilesCommunity.Files
+PackageVersion: 3.3.0.0
+InstallerType: msix
+MinimumOSVersion: 10.0.19041.0
+InstallModes:
+- silent
+- silentWithProgress
+UpgradeBehavior: install
+Dependencies:
+  ExternalDependencies:
```

Comment:
```suggestion
  PackageDependencies:
```
From the [Schema](https://github.com/microsoft/winget-pkgs/blob/master/doc/manifest/schema/1.6.0/installer.md), the other types aren't quite implemented yet. 

Sorry for the extra messaging - I worked on this earlier, and on returning I keep finding things I forgot.

Reply:
When doing the above suggestions I get the following:

![image](https://github.com/microsoft/winget-pkgs/assets/81376423/f0908842-4d7e-46ac-927d-73ba16ad0a35)